### PR TITLE
:zap: Add Summary for Imagen

### PR DESCRIPTION
**Pull request by Tanishq-01**

### CODE REVIEW

**Comment by praeclarumjj3** [comment]
@Tanishq-01, please squash all the commits into one as instructed in the contributing guidelines.

**Comment by kumar-devesh** [comment]
Please pass the text through Grammarly once. Mention the authors of the paper and update the readme.

**Comment by Tanishq-01** [comment]
> Please pass the text through Grammarly once. Mention the authors of the paper and update the readme.

Done

**Comment by Tanishq-01** [comment]
> @Tanishq-01, please squash all the commits into one as instructed in the contributing guidelines.

Done

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ This repo houses summaries for various excitng works in the field of **Deep Lear
 
 - #### Human-level play in the game of Diplomacy by combining language models with strategic reasoning [[Paper](https://www.science.org/doi/epdf/10.1126/science.ade9097)][[Review](./summaries/CICERO.md)]
 
+- #### Photorealistic Text-to-Image Diffusion Models with Deep Language Understanding [[Paper](https://arxiv.org/abs/2205.11487)][[Review](./summaries/imagen.md)]
+
```
Comment (praeclarumjj3) [inline README.md:31]: Please add author names and publication venue here.

**Review by praeclarumjj3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tanishq-01) [reply README.md:31]: Done

File: README.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ This repo houses summaries for various excitng works in the field of **Deep Lear
...
+  - Chitwan Saharia, William Chan, Saurabh Saxena, Lala Li, Jay Whang, Emily Denton, Seyed Kamyar Seyed Ghasemipour, Burcu Karagol Ayan, S. Sara Mahdavi, Rapha Gontijo Lopes, Tim Salimans, Jonathan Ho, David J Fleet, Mohammad Norouzi, **NIPS 2022**
+
+
```
Comment (praeclarumjj3) [inline README.md:34]: Why is an extra line added here?

**Review by praeclarumjj3 — CHANGES_REQUESTED** [review]

**Comment by praeclarumjj3** [comment]
@Tanishq-01 please rebase your branch with the `master` branch:
```.bash
git fetch upstream
git rebase upstream/master
```

File: README.md (hunk 2 above)
Comment (Tanishq-01) [reply README.md]: > Please review now. I guess it is all fine now

**Review by kumar-devesh — APPROVED** [review]
Looks good to me

**Review by praeclarumjj3 — APPROVED** [review]

**Pull request merged**